src/components/product/Disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactNode, MouseEvent } from 'react';
+import { Alert, Button } from 'antd';
+
+const btnLinkStyle = {
+  padding: 0,
+  margin: 0,
+  height: 'auto',
+  lineHeight: 'normal',
+};
+
+export const Disclaimer = ({
+  message,
+  href = '/disclaimer',
+  onDisclaimerClick,
+}: {
+  message?: ReactNode;
+  href?: string;
+  onDisclaimerClick?: (e: MouseEvent) => void;
+}) => {
+  // if the href is internal, we don't want to open it in a new tab
+  const isInternalLink = href && href.startsWith('/');
+
+  return (
+    <Alert
+      type="info"
```

Comment:
Few things:
- info should really inherit a blue colour – I would imagine the pruple to show if there was a "primary" type
- if you can adjust so that it shows blue, I think Info is correct

Reply:
Well I think the link should inherit the primary colour, but info should inherit a blue colour scheme by default.

Reply:
Hmm OK – I think antd is v unconventional in that regard. Yes let's go with Warning instead of Info then.